miden/Cargo.toml

```diff
@@ -25,6 +25,8 @@ path = "src/lib.rs"
 bench = false
 doctest = false
 
+# Note: the `internal` feature must be set when running benchmarks, i.e.
+# `cargo bench --features internal`
 [[bench]]
 name = "program_execution"
 harness = false
```

Comment:
Could we not specify these requirements via `required-features` attribute? (or this doesn't work for benchmarks?)

Reply:
Ah didn't know about that field - added it. But this means that if we run `cargo bench`, we simply won't run `program_execution`.

Should we just get rid of the `internal` feature, and document that everything in the `internal/` module is "not part of public API" (i.e. if someone uses it, they can expect breaking changes even on patch releases)?